### ISSUE TITLE
Screen randomly blacking out

### ISSUE THREAD

**Issue by type2diabetes** [Selbi182/SpotifyBigPicture#34]
So I'm running this off one of my older PCs for a 24/7 mostly unattended live stream and it works great, for the most part. Only slightly annoying thing is sometimes at seemingly random intervals screen goes black before coming back after x seconds, but two times it took minutes so I made it autorefresh every minute but it still happens as shown in [this clip](https://www.youtube.com/watch?v=lDDKzgQMi-0)

Any ideas or possible solutions?

[![black](https://imgur.com/tspOkS4.jpg)](https://youtu.be/lDDKzgQMi-0?t=09 "black")

**Comment by Selbi182** [comment]
Hmm, my first guess be that idle mode is wonky on your device. Can you take a look in the web console when it happens next time? If it's indeed the idle mode, it should say so in there.

It's a bit hard to say what's going on here without additional context :/

**Comment by type2diabetes** [comment]
Idle mode was one of the first things I disabled when I started using it.

Updated java from 19 to 21 which did nothing, still happened. However, it seems I was just being dumb and trusting the browser source in OBS too much, changed it to run in an actual Firefox browser and just window capture it and I havent noticed the issue for hours now having it up on a second monitor. Probably should've done this from the start lol, sorry. :)
